schema/db_migration.sql

```diff
@@ -0,0 +1,212 @@
+-- ids:
+--  id: bigserial
+--  idx: bytea # 16 bytes, indexed, used for lookup
+--  remainder: bytea # remainder of binary data
+--  type: smallint # 0=eoa 1=contract 2=txhash 3=blockhash
+-- TAMCHAU:
+--   "type" is postgres internal key, should use other name -> "id_type".
```

Comment:
Per https://www.postgresql.org/docs/current/sql-keywords-appendix.html type is non-reserved keywork
so it can be used as column without issue. Your concern is valid but it might cause non-backward compatible
with existing applications.